prac_05/wimbledon.py

```diff
@@ -2,27 +2,41 @@
 Wimbledon
 CP1404/CP5632 Practical
 Estimate: 60 minutes
+Actual: 80 minutes
+A program to load data from csv and print it neatly.
 """
 
 
 def main():
+    """Sort data from csv and display data neatly."""
     champion_to_count = {}
     data = load_file("wimbledon.csv")
```

Comment:
Here you let data = file contents, but later you have a function that loads and returns data from filename. Maybe data = get_data(FILENAME) would've been better.